src/mcm_chapter_9/mcm_ch9_answers.markdown

```diff
@@ -0,0 +1,50 @@
+# MCM Chapter 9 Answers
+
+**Q1**: The top of the Exception hierarchy is `Throwable`
+
+**Q2**: Try wraps a block of code which can throw an error. Catch is defined directly after the try and catches specified errors thrown by the block surrounded by try{}
+
+Used like this:
+
+````java
+	try{
+		//Do something that throws and exception
+	}
+	catch(Exception e){
+		//Catch all exceptions
+	}
+````
+
+**Q3**: It doesn't have a try block, so won't catch the error. In fact I'm not sure it'll even compile with no try block, but even if it does - its bad news.
+
+**Q4**: If an exception is not caught then a runtime error will occur and the Java program will stop. Exception handling allows for graceful failure to avoid the program crashing.
+
+**Q5**: Exception "B" will never be caught as it is a subclass of "A" and since "A" is caught first any instance of "B" will be caught by the catch for "B". The fix for this is to change the ordering, so that "B" is caught first.
```

Comment:
I'm a little _confused_.  'B' exceptions will be caught - they will just be caught by the 'A' catch block because it is defined before the 'B' catch block. As things stand - 'B' exceptions will **not** be caught by the catch for 'B'.  

The fix is correct though - the ordering needs to be switched :+1: